src/app/shared/services/http.service.ts

```diff
@@ -133,9 +133,9 @@ export class HttpService {
       .pipe(catchError(this.handleError()));
   }
 
-  updateReport(reportId: string, params: Report, storage: string): Observable<void> {
+  updateReport(reportId: string, body: Report, storage: string): Observable<void> {
     return this.http
-      .post('api/report/' + storage + '/' + reportId, params)
+      .post('api/report/' + storage + '/' + reportId, body)
```

Comment:
String template, doesn't the linter work?

Reply:
Not a rule in the linter yet

Reply:
Well theres a lot more places where this lint rule will need to be applied, I'd rather do it all in a single PR, Just review the actual changes of this PR please. Just keep in mind that a lot still needs to be refactored

Reply:
I'm not asking to rewrite a whole method. While you held this code in your hands, you might as well could've changed it.

Yes, there is a lot to refactor, so why not do as much as possible?

Reply:
https://github.com/wearefrank/ladybug-frontend/issues/462